GestureHandler.js

```diff
@@ -53,6 +54,17 @@ UIManager.RCTView.directEventTypes = {
   },
 };
 
+let shouldHandleInspection = false;
+// Toggled inspector block touches events in order to allow inspecting on Android
+// As event emitter could be hooked only on toggling an inspector
+// it's impossible to allow touches on disabling "Inspect" without toggling inspector
+if (__DEV__ && Platform.OS === 'android') {
```

Comment:
How about mirroring a check for `!global.__RCTProfileIsProfiling` here just as they do [there](https://github.com/facebook/react-native/blob/e7391438093cd5dd5033204cdce62e66509e66e1/Libraries/ReactNative/AppContainer.js#L56) before creating a listener for inspector toggling event?

Reply:
do you think it's a good idea not to "block" GHs if profiler is enabled?

Reply:
Check the link I provided in previous comment: https://github.com/facebook/react-native/blob/e7391438093cd5dd5033204cdce62e66509e66e1/Libraries/ReactNative/AppContainer.js#L56
In React Native they don't create a listener for `'toggleElementInspector'` if profiler is enabled. This line was introduced in this commit: https://github.com/facebook/react-native/commit/28c07f8698fcc39a4727ddec2c1b909a1b1acf39
It seems that React Native explicitly disallow Inspector to be toggled when profiler is enabled. So it would be sensible to add similar check here, so RNGH wouldn't listen to the event that would never be emitted when profiler is enabled.

Reply:
ok, fine